src/inline-stack-switcher.vala

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+public class Graphs.InlineStackSwitcher : Gtk.Widget {
```

Comment:
I'm not sure if it is better to define it this way or put it in namespace Graphs here, but I don't have a strong preference

Reply:
I basically just adopted the Widget from [Elastic](https://gitlab.gnome.org/World/elastic/-/blob/main/src/inline-stack-switcher.vala?ref_type=heads), hence the way this was defined. But this is a fair point, I prefer put it in the namespace as that's more consistent with the rest of the code.